tests/sku-init/sku-init.test.ts

```diff
@@ -3,6 +3,7 @@ import path from 'node:path';
 import fs from 'node:fs/promises';
 import { spawnSync } from 'node:child_process';
 import { runSkuScriptInDir } from '@sku-private/test-utils';
+import skuPackageJson from '../../packages/sku/package.json' with { type: 'json' };
```

Comment:
The `tests` package doesn't install `sku` as a dependency so I used a relative path here.